### PR TITLE
Add additional functionality to the integration benchmark perf test

### PR DESCRIPTION
**Pull request by tylerbenson**
Include agent analysis as well as memory and CPU burn.

### CODE REVIEW

**Comment by tylerbenson** [comment]
@realark here are the results of this test I ran on my laptop:
```shell
./run-perf-test.sh jetty-perftest/build/libs/jetty-perftest-all.jar NoAgent ../build/libs/dd-java-agent.jar
```
[perf_results.csv.txt](https://github.com/DataDog/dd-trace-java/files/1836232/perf_results.csv.txt)

**Comment by realark** [comment]
@tylerbenson  The General approach looks solid.

**Comment by tylerbenson** [comment]
@realark I think this is ready for final review.

**Comment by tylerbenson** [comment]
Another round of results:


Client Version | 1MS Latency | 1MS Throughput | 5MS Latency | 5MS Throughput | 10MS Latency | 10MS Throughput | Agent CPU Burn | Server CPU Burn | Agent RSS Delta | Server Max RSS
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
NoAgent | 1.14ms | 2729.89 | 5.14ms | 586.04 | 10.11ms | 296.45 | 10 | 63007 | 308 | 149270528
0.5.1-SNAPSHOT~08c3e40c | 1.31ms | 2657.47 | 5.72ms | 570.39 | 10.28ms | 292.99 | 792 | 62968 | 17120 | 332226560
Percent Increase | 13.9% | -2.7% | 10.7% | -2.7% | 1.7% | -1.2% | 195.0% | -0.1% | 192.9% | 76.0%

File: dd-java-agent/benchmark-integration/run-perf-test.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/local/bin/bash
```
Comment (realark) [inline dd-java-agent/benchmark-integration/run-perf-test.sh]: Does this have to be `/usr/local/bin/bash`? That won't work on linux. Maybe `/bin/bash`?

Comment (tylerbenson) [reply dd-java-agent/benchmark-integration/run-perf-test.sh]: Done.  I'm guessing there will be other problems around the options I'm using for `time` etc, but I fixed this.

**Pull request merged**